src/dapps/rootstock-collective-dapp/handlers/helpers/utils.js

```
@@ -0,0 +1,38 @@
+export const isAmountValid = (amount, min, max) => {
+  const val = parseFloat(amount);
+
+  if (!isNaN(val) && val >= min && val <= max) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)